docs/HowToGuides/GettingStarted.md

```diff
@@ -254,25 +254,16 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
          --sccache --release-debuginfo --swift-disable-dead-stripping \
          --xcode
        ```
-       **Note:** Building `--xcode` together with `--test` is a common source of issues. So to run
-       tests is recommended to use `ninja` because is normally more stable.
    - Linux (uses Ninja):
      ```sh
-     utils/build-script --release-debuginfo --test --skip-early-swift-driver \
+     utils/build-script --release-debuginfo --skip-early-swift-driver \
        --skip-early-swiftsyntax
      ```
    This will create a directory
    `swift-project/build/Ninja-RelWithDebInfoAssert`
    (with `Xcode` instead of `Ninja` if you used `--xcode`)
    containing the Swift compiler and standard library and clang/LLVM build artifacts.
-   - If the build succeeds: Once the build is complete, the tests will run.
-     - If the tests are passing: Great! We can go to the next step.
-     - If some tests are failing:
-       - Consider [filing a bug report](https://swift.org/contributing/#reporting-bugs).
-       - Note down which tests are failing as a baseline. This baseline will be
-         handy later when you run the tests after making a change.
-   - If the build fails:
-     See [Troubleshooting build issues](#troubleshooting-build-issues).
+   If the build fails, see [Troubleshooting build issues](#troubleshooting-build-issues).
```

Comment:
We have a well placed [section about running tests](https://github.com/apple/swift/blob/eee197cdf153f4fe3c5453659e59abf98045d3c3/docs/HowToGuides/GettingStarted.md#running-tests) following the *Editing code* section. Except we don’t mention the build-script option specifically, which IMO is for the better anyway, because is requires you to get all the build options right in addition to the `--test` flag (*again*).

Reply:
Yeah, at this point the contributor presumably has not gotten to the point of adding their new test case. I agree that it's current position after "Editing the code" is the right place for those instructions, and there's a table of contents at the top in case folks are specifically looking for those instructions.

Reply:
Sounds good to me! :)